resources.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: resources
+title: Resources
+permalink: /resources/
+weight: 20
+sections:
+  - resources:
+    - name: GitHub
+      link: https://github.com/indico/indico
+      logo: github.png
+      summary: Visit our GitHub project site. You can report issues there.
+    - name: Community forum
+      link: https://talk.getindico.io
+      logo: discourse.png
+      summary: Join other Indico administrators on our forum to seek advice or discuss your ideas
+    - name: Getting started
+      link: https://docs.getindico.io
+      logo: indico.png
+      summary: Learn how to set up an Indico instance or discover how to use the Indico API
+    - name: User's guide
+      link: https://learn.getindico.io
+      logo: indico.png
+      summary: Discover all Indico features with video walkthroughs and tutorials
+    - name: Contact
+      link: https://getindico.io/contact
+      logo: indico.png
+      summary: Get in touch with indico developers and maintainers of CERN instance
+    - name: About us
+      link: https://getindico.io/about
+      logo: indico.png
+      summary: Learn more about Indico team members and collaborators
+  - title: Plugins
+    resources:
+    - name: Vidyo
+      link: https://github.com/indico/indico-plugins/tree/master/vc_vidyo
+      logo: github.png
+      summary: Integrate videoconferencing tool with your Indico meetings
+    - name: Paypal payment
+      link: https://github.com/indico/indico-plugins/tree/master/payment_paypal
+      logo: github.png
+      summary: Let your meetings attendees handle their conference fees through Indico
+    - name: Chat
```

Comment:
dead/unmaintained and moved to the attic; let's not advertise it

maybe replace it with the manual payment one (but good luck finding a good way to describe that one) if you need an even amount of plugins in the list